### PR TITLE
pylint aria.storage

### PR DESCRIPTION
**Pull request by mxmrlv**

### CODE REVIEW

File: aria/storage/__init__.py [hunk 1]
```diff
@@ -99,12 +102,16 @@ def __getattr__(self, table):
         return super(ModelStorage, self).__getattr__(table)
 
     def register(self, model_cls):
-        # todo: add documentation
+        """
+        Registers the model type in the resource storage manager.
+        :param model_cls: the model to register.
+        :return:
```
Comment (ran-z) [inline aria/storage/__init__.py]: remove return line

File: aria/storage/__init__.py [hunk 2]
```diff
@@ -353,6 +369,11 @@ def __init__(self, driver, resources=(), **kwargs):
         super(ResourceStorage, self).__init__(driver, resources, **kwargs)
 
     def register(self, resource):
+        """
+        Registers the resource type in the resource storage manager.
+        :param resource: the resource to register.
+        :return:
```
Comment (ran-z) [inline aria/storage/__init__.py]: remove return line

File: aria/storage/drivers.py [hunk 3]
```diff
@@ -29,7 +29,8 @@
 
 import os
 import shutil
-import distutils.dir_util
+# pylint has an issue with distuls and virtualenvs: https://github.com/PyCQA/pylint/issues/73
```
Comment (ran-z) [inline aria/storage/drivers.py]: distuls --> distutils?

File: aria/storage/models.py [hunk 4]
```diff
@@ -123,11 +134,12 @@ def __lt__(self, other):
 
         if self.action != other.action:
             if self.action == 'remove':
-                return True
```
Comment (ran-z) [inline aria/storage/models.py]: why was this necessary? what was wrong about the previous implementation?

File: aria/storage/models.py [hunk 5]
```diff
@@ -158,15 +173,18 @@ class DeploymentUpdate(Model):
     deployment_id = Field(type=basestring)
     state = Field(type=basestring, choices=STATES, default=INITIALIZING)
     deployment_plan = Field()
-    deployment_update_nodes = Field(default=None)
-    deployment_update_node_instances = Field(default=None)
+    depup_nodes = Field(default=None)
```
Comment (ran-z) [inline aria/storage/models.py]: why this change?

File: aria/storage/models.py [hunk 6]
```diff
@@ -251,13 +278,21 @@ class Node(Model):
     max_number_of_instances = Field(type=int)
 
     def relationships_by_target(self, target_id):
+        """
+        Retreives all of the relationship by target.
+        :param target_id: the node id of the target  of the relationship
+        :return:
```
Comment (ran-z) [inline aria/storage/models.py]: return statement's empty; since you're yielding it probably requires a different syntax.
i know google's syntax has a ":yields:" keyword, but since we haven't settled on a format yet i dont know if it should be used or simply have the return removed..

File: aria/storage/models.py [hunk 7]
```diff
@@ -297,13 +335,21 @@ class NodeInstance(Model):
     scaling_groups = Field(default=())
 
     def relationships_by_target(self, target_id):
+        """
+        Retreives all of the relationship by target.
+        :param target_id: the instance id of the target  of the relationship
+        :return:
```
Comment (ran-z) [inline aria/storage/models.py]: same as the previous comment

File: aria/storage/models.py [hunk 8]
```diff
@@ -297,13 +335,21 @@ class NodeInstance(Model):
     scaling_groups = Field(default=())
 
     def relationships_by_target(self, target_id):
+        """
+        Retreives all of the relationship by target.
+        :param target_id: the instance id of the target  of the relationship
```
Comment (ran-z) [inline aria/storage/models.py]: double space

File: aria/storage/structures.py [hunk 9]
```diff
@@ -48,6 +48,10 @@
 
 
 def uuid_generator():
+    """
+    wrapper function which generates ids
+    :return:
```
Comment (ran-z) [inline aria/storage/structures.py]: empty return

File: aria/storage/structures.py [hunk 10]
```diff
@@ -206,6 +219,10 @@ def __eq__(self, other):
 
     @property
     def fields(self):
+        """
```
Comment (ran-z) [inline aria/storage/structures.py]: same as before; plus no capital letter

File: aria/storage/structures.py [hunk 11]
```diff
@@ -174,6 +181,12 @@ class IterPointerField(IterField, PointerField):
 
 
 class Model(object):
+    """
+    Base class for all of the storage models.
+    """
+    # Setting id to avoid missing class member is usage
+    id = None
```
Comment (ran-z) [inline aria/storage/structures.py]: should take the comment off. if it makes sense for the "id" to be here then it makes sense, that's it.

where is it used in the base class anyway?

File: aria/storage/models.py (hunk 4 above)
Comment (mxmrlv) [inline aria/storage/models.py]: too many return statements

File: aria/storage/models.py (hunk 5 above)
Comment (mxmrlv) [inline aria/storage/models.py]: well, it was to long before we switched to 50, change back?
